source/client-side-operations-timeout/tests/runCursorCommand.yml

```diff
@@ -0,0 +1,318 @@
+      - name: createCollection
+        object: *db
+        arguments:
+          collection: *cappedCollection
+          capped: true
+          size: 4096
+          max: 3
+        saveResultAsEntity: *cappedCollection
+      - name: insertMany
+        object: *cappedCollection
+        arguments:
+          documents:
+            - { _id: 1, x: 11 }
+            - { _id: 2, x: 22 }
+      - name: createCommandCursor
+        object: *db
+        arguments:
+          commandName: find
+          command: { find: *cappedCollection, batchSize: 1, tailable: true }
+          timeoutMode: iteration
+          timeoutMS: 20
+          batchSize: 1
+          cursorType: tailable
+        saveResultAsEntity: &tailableCursor tailableCursor
+      # Iterate the cursor twice: the first iteration will return the document from the batch in the find and the
+      # second will do a getMore.
+      - name: iterateUntilDocumentOrError
+        object: *tailableCursor
+      - name: iterateUntilDocumentOrError
+        object: *tailableCursor
+        expectError:
+          isTimeoutError: true
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              commandName: drop
+          - commandStartedEvent:
+              commandName: create
+          - commandStartedEvent:
+              commandName: insert
+          - commandStartedEvent:
+              commandName: find
+              databaseName: *db
+              command:
+                find: *cappedCollection
+                tailable: true
+                awaitData: { $$exists: false }
+                maxTimeMS: { $$exists: false }
+          - commandStartedEvent:
+              commandName: getMore
+              databaseName: *db
+              command:
+                getMore: { $$type: ["int", "long"] }
+                collection: *cappedCollection
+                maxTimeMS: { $$exists: false }
+
+  # The timeoutMS value should be refreshed for getMore's. This is a failure test. The find inherits timeoutMS=10 from
+  # the collection and the getMore blocks for 15ms, causing iteration to fail with a timeout error.
+  - description: Tailable cursor awaitData iteration timeoutMS is refreshed for getMore - failure
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: *failPointClient
+          failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+              failCommands: ["getMore"]
+              blockConnection: true
+              blockTimeMS: 15
+      - name: dropCollection
+        object: *db
+        arguments:
+          collection: &cappedCollection cappedCollection
+      - name: createCollection
+        object: *db
+        arguments:
+          collection: *cappedCollection
+          capped: true
+          size: 4096
+          max: 3
+        saveResultAsEntity: *cappedCollection
+      - name: insertMany
```

Comment:
Should this operation insert data? Without inserting data, the Go Driver gives this error:

```
unified_spec_runner.go:134: error running operation "insertMany" at index 3: execution failed: too few bytes to read next component
```
Adding documents resolves this. Here is an example: https://github.com/mongodb/mongo-go-driver/pull/1273/files#diff-673935c538ceedaf9c7d83fa37b5d5f038f634149c29600d7120b8d9813b414aR511